utils/goDotEnv.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func GoDotEnvVariable(key string) string {
+
+	err := godotenv.Load(".env")
+
+	if err != nil {
+		log.Println("Error loading .env file")
+	}
+
+	return os.Getenv(key)
```

Comment:
no need to return anything from this function

As the loaded variables can be directly accessed in code via os.Getenv() or os.LookupEnv()